static/app/components/smartSearchBar/searchDropdown.tsx

```diff
@@ -629,3 +631,7 @@ const Value = styled('span')<{hasDocs?: boolean}>`
 
   ${p => p.theme.overflowEllipsis};
 `;
+
+const IconOpenWrapper = styled('div')`
```

Comment:
You should be able to do `styled(IconOpen)` here and not need an extra div

Reply:
That's a better idea, updated!